pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -121,6 +121,10 @@ func (lvm *pmemLvm) CreateDevice(name string, size uint64, nsmode string) error
 				if err != nil {
 					return err
 				}
+				err = WaitDeviceAppears(device)
```

Comment:
Is this change really required?. If I understood correctly, the issue wasn't with missing /dev mounting in LVM mode?.

Reply:
I think it is needed. The "wait for device" is different issue from "missing /dev mount". My observations have shown that creating of LVolume and following appearance of it's entry in /dev/ is not synchronous operation:
after lvcreate returns and driver code goes on accessing /dev/ entry,
the new device entry either exists in /dev, or appears shortly later.
As it may appear asynchronously after short delay, it makes sense to retry few times instead of failing on 1st try.
With failing we destroy chance to create volume which really was just showing up with a delay.
The combination of lvcreate() + WaitDeviceAPpears() is kind an attempt to make the creation operation synchronous by semantics.

Reply:
@okartau Thanks for the clarification.